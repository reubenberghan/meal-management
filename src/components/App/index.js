import * as React from 'react'

export default function App () {
  return <div><h1>Meal Management App</h1></div>
}