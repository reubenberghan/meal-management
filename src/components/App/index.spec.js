import * as React from 'react'
import { shallow } from 'enzyme'

import App from '.'

describe('components:App', () => {
  it('should render the App component and CSS properties properly', () => {
    expect(toJson(shallow(<App />))).toMatchSnapshot()
  })
})
