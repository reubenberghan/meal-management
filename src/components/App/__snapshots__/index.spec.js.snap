// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:App should render the App component and CSS properties properly 1`] = `
<div>
  <h1>
    Meal Management App
  </h1>
</div>
`;
